js/private/js_image_layer.bzl

```
@@ -18,9 +18,19 @@ load("@bazel_skylib//lib:paths.bzl", "paths")
 
 _DOC = """Create container image layers from js_binary targets.
 
-js_image_layer supports transitioning to specific platform for cross-compiling.
+By design, js_image_layer doesn't have any preference over rule which assembles the container image. 
+However, this comes with a few quirks; the downstream rule, `oci_image` or `container_image` in this case,
```

```suggestion
This means the downstream rule (`oci_image`, or `container_image` in this case) must set a proper `workdir` and `cmd` to for the container work.
```
? 🤷 

What do you mean by "in this case" though? The example below? Or `js_image_layer`? You could probably say "in this example" or "with js_image_layer"?

---

the downstream rule, it should be `in most cases`